source/server/getting-started/configuration/index.rst

```diff
@@ -2,81 +2,104 @@
 Configuring Sponge
 ==================
 
-You can find all configuration files inside the "config" folder.
-
-Config Syntax
-=============
-
-Most configuration files will make use of the :doc:`HOCON <hocon>` format.
-
 .. toctree::
     :maxdepth: 3
     :titlesonly:
 
+    sponge-conf
     hocon
-    json
 
-What You Can Configure
-======================
+.. _configuration_sponge:
 
-.. toctree::
-    :maxdepth: 3
-    :titlesonly:
+This article is about configuring Sponge itself, plugins will usually create their own configuration files inside
+the servers `config` directory.
+For Vanilla configuration check out the `Minecraft Wiki <https://minecraft.gamepedia.com/Server.properties>`__.
 
-    sponge-conf
-    server-properties
+Sponge Configuration Files
+==========================
+
+Sponge has multiple configuration files which all can be found in the ``config/sponge/`` directory of your server:
+
+* ``global.conf``
+  and its dimension and world subconfigs in the ``worlds`` folder
+* ``tracker.conf``
```

Comment:
Maybe have these two above the worlds folder mentioned because it may cause confusion if people understand there may be a `tracker.conf` and `custom_dat.conf` for each world folder.